content/chapters/compute/lab/quiz/parent-faults-before-fork.md

```diff
@@ -0,0 +1,12 @@
+# Parent Faults before `fork()`
+
+## Question Text
+
+What causes the page faults that occur between the first and second steps?
+
+## Question Answers
+
+- Calling `fork()` duplicates the pages previously allocated by the parent
++ Demand paging makes the pages in the `p` array to be mapped to frames only when written.
+- The OS duplicates the parent's pages in preparation for `fork()`
+- `mmap()` sets the pages to be mapped at a later time, decided by the OS
```

Comment:
```suggestion
- `mmap()` sets the pages to be mapped at a later time, decided by the OS

## Feedback

```
Add feedback for this quiz.

Reply:
I wanted to, but I don't know what to write. The feedback is kind of the answer itself.

Reply:
We could leave a link to a section where demand paging is explained. If that seems ugly, no feedback is also ok.

Reply:
It's ugly and a bit unnecessary. Reading the lab section on demand paging is kind of the default feedback. I'd leave this question without feedback.